src/index.js

```diff
@@ -311,7 +312,16 @@ function createConfig(options, entry, format, writeMeta) {
 		return globals;
 	}, {});
 	if (options.globals && options.globals !== 'none') {
-		globals = Object.assign(globals, parseGlobals(options.globals));
+		globals = Object.assign(globals, parseMappingArgument(options.globals));
+	}
+
+	const NODE_ENV = process.env.NODE_ENV || 'production';
```

Comment:
this is for libraries

Reply:
so I don't support having that as the default at this point in the "toolchain"

Reply:
That is a good point. `NODE_ENV` and `process.env.NODE_ENV` are probably best left undefined altogether by default.

Reply:
@ForsakenHarmony think it's better to not do inlining by default?  I'm good with that.